xbmc/Application.cpp

```diff
@@ -3618,6 +3618,14 @@ void CApplication::OnPlayBackStopped()
   g_windowManager.SendThreadMessage(msg);
 }
 
+void CApplication::OnPlayBackError()
+{
+  //@todo Playlists can be continued by calling OnPlaybackEnded instread
```

Comment:
spelling: `instead`

Reply:
should I add another todo to correct he spelling of the first one? :)

Reply:
just kidding